src/LinuxTracing/PerfEventReaders.cpp

```diff
@@ -267,7 +267,7 @@ MmapPerfEvent ConsumeMmapPerfEvent(PerfEventRingBuffer* ring_buffer,
   size_t filename_size =
       header.size - filename_offset - sizeof(RingBufferSampleIdTidTimeStreamidCpu);
   std::vector<char> filename_vector(filename_size);
```

Comment:
Out of scope, but we should switch to `auto filename_chars = std::make_unique<char[]>(filename_size)` (not `for_overwrite` because we want the initialization to zero), because we just need a chunk of memory to fill.

Reply:
Same below.

Reply:
I agree. But this change does not really fit into this PR. Could you take care of this?